client/rest-high-level/src/main/java/org/elasticsearch/client/RequestConverters.java

```diff
@@ -131,14 +131,14 @@
 import java.util.StringJoiner;
 import java.util.stream.Collectors;
 
-final class RequestConverters {
-    static final XContentType REQUEST_BODY_CONTENT_TYPE = XContentType.JSON;
+public final class RequestConverters {
```

Comment:
we should add javadocs to this class if it becomes public

Reply:
But, yeah, this class should have javadocs. And the actual public methods should shift to the top, but maybe that is a nice, small, mechanical change to do in a follow up PR.